wigwam/_utils.py

```diff
@@ -203,6 +203,24 @@ def test_image(image: Image, expression: str) -> bool:
             raise
 
 
+def get_libdir(
+    base_tag: str,
+) -> str:
+    """Determine if the given image uses `lib64` or `lib` as its `LIBDIR`."""
+    # Unlike with the CMake Install step, `libdir` can be checked directly on this image
+    # because it has something at $INSTALL_PREFIX. Check the base tag for lib64 or lib.
+    with temp_image(base_tag) as temp_img:
+        uses_lib64 = test_image(image=temp_img, expression='"$INSTALL_PREFIX/lib64"')
+
+    if uses_lib64:
+        libdir: str = "lib64"
+    else:
+        libdir = "lib"
+        assert test_image(image=temp_img, expression='"$INSTALL_PREFIX/lib"')
```

Comment:
`temp_img` no longer exists in this scope. It's created in the `with` block above on line 212 and should get deleted when we exit that block.